### ISSUE TITLE
Evaluation results are always zero 

### ISSUE THREAD

**Issue by aravindsairam** [hellojialee/Improved-Body-Parts#36]
Average precision and average recall from evaluate.py always give zero. 

![Screenshot 2021-01-10 at 11 43 27](https://user-images.githubusercontent.com/34033592/104120745-94d28d00-5339-11eb-9d3f-29b278a3c0e1.png)

I am using images of size 512(other parameters are default as it is), I have trained the model without checkpoint and for 20 epochs with COCO dataset. The results were decent. But evaluation is zero(evaluation on validation set for 50 ids). What would be the reason ?  

Thanks, 
Aravind

**Comment by hellojialee** [comment]
Hi @aravindsairam 
I suggest you run demo_image.py to check that the output or the file format is correct.
